cocode/src/components/Common/Toast/index.js

```diff
@@ -0,0 +1,26 @@
+import ReactDOM from 'react-dom';
+import React from 'react';
+import ToastContainer from './ToastContainer';
+
+function addToast(message, options) {
+	const toastElement = document.getElementById('toast-root');
+
+	const newToast = {
+		id: Math.random(),
+		type: options.type || 'info',
+		message,
+		...options
+	};
+
+	ReactDOM.render(<ToastContainer toast={newToast} />, toastElement);
```

Comment:
여기가 일반적인 컴포넌트처럼 바뀌겠죠
```suggestion
	return (<ToastContainer toast={newToast} />, toastElement);
```

Reply:
리뷰어님 귀한 시간 내셔서 코드 리뷰해주셔서 감사합니다! 
직접 `ReactDOM.render`를 호출하는 대신 App.js에서 Toast를 그려주는 방식을 찾아봐야겠어요! 
남은 한 주도 화이팅하세요 😁